src/PKSim.Core/Snapshots/Mappers/CompoundMapper.cs

```diff
@@ -106,35 +109,66 @@ private async Task updateAlternatives(ModelCompound compound, Alternative[] snap
 
       private void updatePkaTypes(ModelCompound compound, SnapshotCompound snapshot)
       {
-         snapshot.PkaTypes?.Each((pkaType, i) =>
+         snapshot.PkaTypes?.Each((pkaType, i) => updatePkaType(compound, pkaType, i));
+         synchronizePkaValueOrigins(snapshot.PkaTypes?.FirstOrDefault(), compound);
```

Comment:
In snapshot we only save what's not neutral. To ensure that we are consistent, I am synchronizing  ALL Pka